kodilla-stream/src/main/java/com/kodilla/stream/StreamMain.java

```diff
@@ -1,17 +1,24 @@
 package com.kodilla.stream;
 
-import com.kodilla.stream.book.Book;
-import com.kodilla.stream.book.BookDirectory;
+import com.kodilla.stream.forumuser.Forum;
+import com.kodilla.stream.forumuser.ForumUser;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.Map;
 import java.util.stream.Collectors;
 
 public class StreamMain {
     public static void main(String[] args) {
-        BookDirectory theBookDirectory = new BookDirectory();
-        String theResultStringOfBooks = theBookDirectory.getList().stream()
-                .filter(book -> book.getYearOfPublication() > 2005)
-                .map(Book::toString)
-                .collect(Collectors.joining(",\n","<<",">>"));
+        Forum forum = new Forum();
+        Map<Integer, ForumUser> selectedForumUsers = forum.getUserList().stream()
+                .filter(forumUser -> forumUser.getSex() == 'M')
+                .filter(forumUser -> ChronoUnit.YEARS.between(forumUser.getDayOfBirth(), LocalDate.now()) >= 20)
+                .filter(forumUser -> forumUser.getPublishedPosts() >= 1)
+                .collect(Collectors.toMap(ForumUser::getIdentityNumber, forumUser -> forumUser));
 
-        System.out.println(theResultStringOfBooks);
+        System.out.println("# elements: " + selectedForumUsers.size());
+        selectedForumUsers.entrySet().stream()
+                .map(ForumUserEntry -> ForumUserEntry.getKey() + ": " + ForumUserEntry.getValue())
```

Comment:
zmienne zaczynamy pisać z malej litery (tak wygląda, jakby to było wywolanie metody statycznej)